src/skan/csr.py

```diff
@@ -1092,3 +1094,121 @@ def make_degree_image(skeleton_image):
                     bool_skeleton.astype(int), degree_kernel, mode='constant'
                     )
     return degree_image
+
+
+def sholl_analysis(
+        skeleton,
+        soma=None,
```

Comment:
I'd call this "center", so that this function remains domain-agnostic.

Reply:
I thought "center" was a bit ambiguous, it could be that it's not the skeleton's geometric center. What about "shell_center"?

Reply:
I prefer center but shell_center is fine.